### ISSUE TITLE
Goodness of fit

### ISSUE THREAD

**Issue by erex** [DistanceDevelopment/dsims#47]
This may be difficult to achieve.  Models are fitted and estimates produced for each simulation replicate, but there is no assessment of model fit.  Likelihoods and AIC scores are produced, but absolute fit is not assessed.  In my particular application, I'd be interested to know how often density estimates are being derived from models that do not fit the data.

I suppose this would mean sending each replicate data set and fitted model off to `gof_ds` and harvesting statistics from there.

**Comment by LHMarshall** [comment]
Given that the Chi-square is better assessed by eye over a number of options for break points for the bins and that it is largely unfeasible to run a bootstrap within a simulation I imagine the best option for this would be to add an option for Cramer-von Mises p-values to be collected and stored?

**Comment by lenthomas** [comment]
Yes I agree ks or CvM better for continuous data (are stuck with chi-sq for discrete data).

**Comment by erex** [comment]
Would be happy if CvM could be included as an option.  You're right, performing KS test (that requires bootstrap) would not be a great idea.

**Comment by lenthomas** [comment]
I don't think a KS test requires a bootstrap (see, e.g., `ks.test` in `R`).  The CvM required a simulation study to produce a lookup table of critical values in the old FORTRAN code, but I think the p-value is now derived some other way in the GoF outputs in `mrds`.  CvM is supposidly more powerful than KS I think, so possibly preferable.  In my experience they rarely differ in terms of inference made on GoF for fitted detection functions.
